package.json

```diff
@@ -37,7 +37,6 @@
     "audit-ci": "^4.0.0",
     "chokidar-cli": "^2.1.0",
     "concurrently": "^6.0.0",
-    "cpx": "^1.5.0",
```

Comment:
I searched online, found this Stackoverflow [answer](https://stackoverflow.com/a/27362322) and tried using `depcheck`. This is what I found.

![image](https://user-images.githubusercontent.com/45748283/199528875-7d981e0f-77e6-436b-9421-7c23e5693853.png)

[`npm-check`](https://github.com/dylang/npm-check#use) also marks the same packages as unused. Have I done it correctly?

Reply:
I don't think any automated ways will be reliable. A lot of these deps are used in scripts but not in code.

I removed `cpx` because it was not used in any scripts and had a vulnerable dependency on `minimatch`.